src/nidigital/examples/burst_with_start_trigger/nidigital_burst_with_start_trigger_grpc.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python
+
+import argparse
+import grpc
+import nidigital
+import os
+import sys
+
+
+def example(resource_name, options, address, port, trigger_source=None, trigger_edge=None, args.address, args.port):
```

Comment:
There are multiple issues with the parameters:
- I don't think optional arguments can come before non-optional ones
- You might *invoke* this with `args.address` but that's not the parameter name.

Reply:
Also, don't forget to link the work item:
[Task 2190911](https://ni.visualstudio.com/DevCentral/_workitems/edit/2190911): Create gRPC NI-Digital example